cmake/CMakeLists.txt

```diff
@@ -135,12 +135,15 @@ endif (protobuf_BUILD_SHARED_LIBS)
 if (MSVC)
   # Build with multiple processes
   add_definitions(/MP)
-  add_definitions(/wd4244 /wd4267 /wd4018 /wd4355 /wd4800 /wd4251 /wd4996 /wd4146 /wd4305 /wd4309)
+  add_definitions(/wd4244 /wd4267 /wd4018 /wd4355 /wd4800 /wd4251 /wd4996 /wd4146 /wd4305 /wd4309 /wd4065 /wd4506 /wd4307 /wd4334)
```

Comment:
could you help sort the whole list? it would be easier to add / lookup in the future.

Reply:
Good call; fixed. Also included descriptions of the suppressed warnings.